website/api/controllers/entrance/signup.js

```diff
@@ -57,6 +59,13 @@ the account verification message.)`,
       description: 'The user\'s last name.',
     }
 
+    // Add sandboxExpirationTimestamp as an optional input
+    // sandboxExpirationTimestamp: {
+    //   required: false,
+    //   type: 'string',
+    //   description: 'An ISO 8601 timestamp of when this user\'s Fleet sandbox will expire'
+    // },
+
```

Comment:
The other dates/times we have so far are JS timestamps right?  If so, then I think this should be the same (ie a number), and be named `sandboxExpiresAt` in terms of how it is persisted in the model.  This can be different from the API.

Reply:
Makes sense!